vmm/modules/suspend/suspend.c

```diff
@@ -85,13 +87,31 @@ static void prepare_s3_percpu(guest_cpu_handle_t gcpu, void *unused UNUSED)
 	}
 }
 
+static void save_sipi_mem(uint64_t sipi_addr)
```

Comment:
the sipi_page is also a member of suspend_data_t, suggest to remove the input parameter.